cmd/client-s3.go

```diff
@@ -251,6 +251,9 @@ func (c *s3Client) AddNotificationConfig(arn string, events []string, prefix, su
 	// Get any enabled notification.
 	mb, e := c.api.GetBucketNotification(bucket)
 	if e != nil {
+		if ignoreExisting {
```

Comment:
We should validate the right error here, we shouldn't ignore all errors

Reply:
done.